config/settings.yml

```diff
@@ -77,3 +77,5 @@ environment:
 
 slack:
   publish_register_alerts_channel: "#twd_publish_register_tech"
+
+track_validation_errors: true
```

Comment:
@darokel Did you wanted it off as default for `development.yml`?

Reply:
Yes I think it should be off by default and enabled explicitly